source/draw/RasterMonochrome.ooc

```diff
@@ -33,6 +33,7 @@ RasterMonochrome: class extends RasterPacked {
 	create: func (size: IntSize2D) -> Image { This new(size) }
 	copy: func -> This {
 		result := This new(this)
+		result _buffer = ByteBuffer new(this _buffer size)
```

Comment:
Problem is, this will have to be added to the other `RasterPacked` classes as well (Bgr, Bgra, Uv`. Maybe what this function does should be moved to a function in `RasterPacked` instead? `copy` looks identical in all of them anyway.